.github/workflows/cicd.yml

```diff
@@ -399,7 +399,41 @@ jobs:
     uses: ./.github/workflows/reusable-test.yml
     with:
       name: user_mode_multi_threaded_stress
-      test_command: .\ebpf_stress_tests_um -jtt=32 -jtd=300
+      test_command: .\ebpf_stress_tests_um -tt=32 -td=10
+      build_artifact: Build-x64
+      environment: windows-2022
+      code_coverage: false
+      leak_detection: false
+      gather_dumps: true
+      capture_etw: true
+
+  # Run multi-threaded stress tests against the kernel mode eBPF sub-system.
+  # The '-td' parameter specifies the run time per test, so the total run-time will be a multiple of the total number
+  # of tests, as each test is run sequentially.
+  kernel_mode_multi_threaded_stress_test:
+    needs: regular
+    if: github.event_name == 'schedule'
+    uses: ./.github/workflows/reusable-test.yml
```

Comment:
Since you're adding reusable-test.yml instances in cicd.yml, I think you need to update the after_n_builds count in codecov.yml

Reply:
As checked in, `.github\codecov.yml` has the `after_n_builds` field set to 23 which already exceeds the new actual count (22) _including_ the new tests added in this PR:

```
D:\wrk\ebpf-for-windows\.github [mt-stress-tests-km]
PS > findstr after_n_builds .\codecov.yml
    after_n_builds: 23 # (reusable-test.yml instances in cicd.yml) * (# configurations per instance)
    after_n_builds: 23 # (reusable-test.yml instances in cicd.yml) * (# configurations per instance)


D:\wrk\ebpf-for-windows\.github [mt-stress-tests-km]
PS > findstr uses .\workflows\cicd.yml | findstr reusable-test.yml | measure-object -line


Lines Words Characters Property
----- ----- ---------- --------
   22
```

What am I missing?

Reply:
You're missing the `* (# configurations per instance)` part.

Some of those uses have 2 configurations (Debug, Release) each, some have 1 (FuzzerDebug).

Reply:
Thanks, Dave. That clears it up.  Now I'm waffling on whether we need to run MT stress on debug _and/or_ release builds as I can argue for both sides :-P (Personal preference being 'release only'.)

I'm not aware of a similar precedence here (or quite possibly overlooking an existent instance) so I'll go with the team consensus. Maybe we could discuss this during tomorrow's meeting?

Reply:
The current value for `after_n_builds` counter in `.github\codecov.yml` is incorrect.  I've filed an [issue](https://github.com/microsoft/ebpf-for-windows/issues/2373) to address this.

Also, only tasks that upload code coverage reports need to be accounted for in that counter and the tasks added in this PR do not do so.  As a result, this PR does not need to update the `after_n_builds` counter.